src/store/profileReducer.js

```diff
@@ -0,0 +1,49 @@
+import { ADD_POST, UPDATE_NEW_POST_TEXT } from './actionTypes';
+
+const initialState = {
+  posts: [
+    {
+      id: '1',
+      message: 'Hi, how are you',
+      likesCount: '12',
+    },
+    {
+      id: '2',
+      message: 'It`s my first posts',
+      likesCount: '7',
+    },
+  ],
+
+  newPostText: '',
+};
+
+const profileReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_POST:
+      const newPost = {
+        id: 7,
+        message: state.newPostText,
+        likesCount: 0,
```

Comment:
Consider creating a constant for this default '0' value.

Reply:
/fxd